observability-kit-test/observability-kit-agent-it/src/main/resources/vaadin-featureflags.properties

```diff
@@ -0,0 +1 @@
+com.vaadin.experimental.expressBuild=true
```

Comment:
It should be removed, as this is the default mode for Vaadin 24 and the feature flag does not exist anymore